Add Indian holidays

notlmn opened the pull request:
More than 90% of holidays in India are dynamic. This PR adds only fixed holidays for now. Need to wait till #2 gets merged to add add dynamic holidays.

WebReflection commented:
Apologies @notlmn but too many things changed in the meanwhile (early state, I want to get this right).

Please have a look at latest state and see the `holidays/.exmple` folder to know more.

Feel free to check the `utils.js` file too and other countries to see how things are retrieved (i.e. the gb or the us are quite complete in potentials)

notlmn commented:
Hey, I am working on it. I also did a rebase locally. This PR should not affect most things considering the changes are in a separate file.

I waited because I was working on how to find dates for holidays, because holidays in India literally depend the position of Sun and Moon.

WebReflection commented:
wow! looking forward to the updated PR then 🎉

notlmn commented:
Yeah, holy whatever.

As I told you already more than 90% of holidays are dynamic (most literally depend on position of Sun and Moon). The one's already statically provided are the only national holidays.

I'm thankful for your patience, because this PR might take some more time because I'm in an overcrowded train that might take another 2 hours to reach it's destination.

WebReflection commented:
I didn't mean to rush you at all ... take all the time you want but I'm sure already it'd be a fantastic contribution (in general, not only for this project).

notlmn commented:
@WebReflection I've rebased and added all the static holidays for now.

It looks like the dynamic holidays might take some more time (I'm planning to work on an Indian calendar that _might_ help solve dynamic dates).

You can merge this for now, I'll make a PR later to remaining dynamic holidays when I'm done.
